src/test/ui/feature-gates/feature-gate-rustdoc_internals.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0658]: `#[doc(keyword)]` is meant for internal use only
```

Comment:
Can you add a test to make sure the "feature gate renamed error" is shown for `#![feature(doc_keyword)]` and `doc_primitive`?